### DISCUSSION TITLE
Project Sponsoring by Shelly?

### DISCUSSION THREAD

**Discussion by Floetenprinz** [mongoose-os-apps/shelly-homekit#1529] [General]
I recently checked for updates regarding matter support for shelly devices and found this project with a shelly plug-in https://github.com/Luligu/matterbridge-shelly. I saw that this project is sponsored by shelly corp. Wouldn't it be an idea that shelly-homekit would also be sponsored by shell corp?  I think it would be worth a try.

**Comment by markirb** [comment]
Any support is appreciated... 

But I don't know if this is politically feasible.
